code/modules/mining/mine_items.dm

```diff
@@ -275,3 +306,121 @@
 		var/area/Z = get_area(A)
 		if(!is_type_in_list(Z, blacklist))
 			L.contents += A
+
+//Pod turfs and objects
+
+
+//Floor
+/turf/open/floor/plasteel/pod
+	name = "pod floor"
+	icon_state = "podfloor"
+	icon_regular_floor = "podfloor"
+
+//Table
+/obj/structure/table/survival_pod
+	icon = 'icons/obj/lavaland/survival_pod.dmi'
+	icon_state = "table"
+	smooth = SMOOTH_FALSE
+
+/obj/structure/fans
+	icon = 'icons/obj/lavaland/survival_pod.dmi'
+	icon_state = "fans"
+	name = "environmental regulation system"
+	anchored = 1
+	density = 1
+
+//Sleeper
+/obj/machinery/sleeper/survival_pod
+	icon = 'icons/obj/lavaland/survival_pod.dmi'
+	icon_state = "sleeper"
+
+/obj/machinery/sleeper/survival_pod/update_icon()
+	if(state_open)
+		overlays.Cut()
+	else
+		overlays += "sleeper_cover"
+
+//Computer
+/obj/item/device/gps/computer
+	name = "pod computer"
+	icon_state = "pod_computer"
+	icon = 'icons/obj/lavaland/pod_computer.dmi'
+	anchored = 1
+	density = 1
+	pixel_y = -16
+
+/obj/item/device/gps/computer/attack_hand(mob/user)
+	attack_self(user)
+
+//Bed
+/obj/structure/bed/pod
+	icon = 'icons/obj/lavaland/survival_pod.dmi'
+	icon_state = "bed"
+
+//Survival Storage Unit
+/obj/machinery/smartfridge/survival_pod
+	name = "survival pod storage"
+	desc = "A heated storage unit."
+	icon_state = "bedcomputer"
+	icon = 'icons/obj/lavaland/donkvendor.dmi'
+	icon_on = "donkvendor"
+	icon_off = "donkvendor"
+	max_n_of_items = 10
+	pixel_y = -4
+
+/obj/machinery/smartfridge/survival_pod/accept_check(obj/item/O)
+	if(istype(O, /obj/item))
+		return 1
+	return 0
+
+/obj/machinery/smartfridge/survival_pod/New()
+	..()
+	for(var/i in 1 to 5)
+		var/obj/item/weapon/reagent_containers/food/snacks/donkpocket/warm/W = new(src)
+		load(W)
+	if(prob(50))
+		var/obj/item/weapon/storage/pill_bottle/dice/D = new(src)
+		load(D)
+	else
+		var/obj/item/device/instrument/guitar/G = new(src)
+		load(G)
+
+//Walls
+
+/turf/closed/wall/shuttle/survival_pod
+	name = "wall"
+	icon = 'icons/turf/walls/survival_pod_walls.dmi'
+	icon_state = "smooth"
+	walltype = "shuttle"
+	smooth = SMOOTH_MORE|SMOOTH_DIAGONAL
+	canSmoothWith = list(/turf/closed/wall/shuttle/survival_pod, /obj/machinery/door/airlock)
```

Comment:
should probably specifically smooth with the pod airlock but not too much of a concern